### PR TITLE
Chore: Setup tests-coverage

### PR DESCRIPTION
**Pull request by Gentille-dev**
- Setup tests and coverage for frontend

### CODE REVIEW

**Comment by Gentille-dev** [comment]
Screenshot 2023-08-31 at 11 44 30

**Review by NdatimanaSamuel — APPROVED** [review]

File: .dockerignore
Comment (Muhet) [inline .dockerignore:1]: add .dockerignore file

**Review by Muhet — CHANGES_REQUESTED** [review]
add dockerignore

File: .dockerignore
Comment (Gentille-dev) [reply .dockerignore:1]: its already there.. please

**Review by Muhet — APPROVED** [review]
well done

**Review by Lydie260 — APPROVED** [review]

**Review by KAYITARES — APPROVED** [review]

File: tests/example.test.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { sum } from "@/utils/math";
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});
+
```
Comment (ceelogre) [inline tests/example.test.ts]: How is this example test different from math.test?

**Review by ceelogre — CHANGES_REQUESTED** [review]
Reply to my comment. Also, it looks like your branch is behind. 
image

File: tests/example.test.ts (hunk 1 above)
Comment (Gentille-dev) [reply tests/example.test.ts]: I was trying to run multiple tests but I will remain with one test.

Comment (Gentille-dev) [reply tests/example.test.ts]: Let me rebase

**Review by ceelogre — APPROVED** [review]

**Pull request merged**